source/scale/high-availability-cluster-based-deployment.rst

```diff
@@ -8,462 +8,414 @@ High availability cluster-based deployment
 
-- LDAP sync
-- Data retention
-- Compliance exports
-- Elasticsearch indexing
+:ref:`Enable RunScheduler <configure/experimental-configuration-settings:run scheduler>` for all servers in the cluster. This setting should remain ``true``. Changing it prevents the leader from scheduling recurring jobs.
+ 
+Plugins
+~~~~~~~~
 
-Make sure you have set ``JobSettings.RunScheduler`` to ``true`` in ``config.json`` for all app and job servers in the cluster. The cluster leader will then be responsible for scheduling recurring jobs.
+Now that the cluster is fully operational, ensure Mattermost plugins are properly deployed and functioning across all nodes.
 
-.. note::
+- Automatic plugin propagation: When adding or upgrading plugins, they are automatically distributed across cluster nodes if shared file storage (e.g., NAS, S3) is in use.
+- File storage: Ensure the :ref:`FileSettings.Directory <configure/environment-configuration-settings:local storage directory>` is a shared NAS location (``./data/``). Failure to do so could corrupt storage or disrupt high availability functionality.
+- Plugin State on reinstallation:
 
-  It is strongly recommended not to change this setting from the default setting of ``true`` as this prevents the ``ClusterLeader`` from being able to run the scheduler. As a result, recurring jobs such as LDAP sync, Compliance Export, and data retention will no longer be scheduled.
+  - v5.14 and earlier: Retains previous Enabled/Disabled state.
+  - v5.15 and later: Starts in a Disabled state by default.
 
-In previous Mattermost Server versions, and this documentation, the instructions stated to run the Job Server with ``RunScheduler: false``. The cluster design has evolved and this is no longer the case.
+mmctl
+~~~~~~
 
-Plugins and High Availability
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+We recommend using :doc:`mmctl commands </manage/mmctl-command-line-tool>` to make changes in a high availability environment because mmctl commands interact through the API layer to notify all nodes of changes, and no server restarts are required.
 
-When you install or upgrade a plugin, it's propagated across the servers in the cluster automatically. File storage is assumed to be shared between all the servers, using services such as NAS or Amazon S3.
+Review cluster status
+---------------------
 
-If ``"DriverName": "local"`` is used then the directory at ``"FileSettings":`` ``"Directory": "./data/"`` is expected to be a NAS location mapped as a local directory. If this is not the case High Availability will not function correctly and may corrupt your file storage.
+Once you've set up new Mattermost servers with identical copies of the configuration, Verify the servers are functioning by hitting each independent server through its private IP address. Restart each machine in the cluster.
```

Comment:
`Verify` should be in lower case. Also, I'm not sure why it says to restart each machine now.